tests/acceptance/features/apiCollaboration/wopi.feature

```diff
@@ -1080,3 +1080,51 @@ Feature: collaboration (wopi)
       """
     And for user "Alice" folder "testFolder" of the space "new-space" should not contain these files:
       | simple.odt |
+
+
+  Scenario Outline: open file with .potx extension and template
+    Given user "Alice" has uploaded file "filesForUpload/template.xltx" to "template.xltx"
```

Comment:
Can we check the MUST templates?
https://github.com/owncloud/ocis/issues/9785#issue-2460826857
>- Must:
>    - `DOTX`
>    - `OTT`

Reply:
Hmm, I think this test should be done in web e2e